src/settings-ui/Settings.UI/ViewModels/MouseUtilsViewModel.cs

```diff
@@ -71,12 +71,14 @@ public MouseUtilsViewModel(ISettingsUtils settingsUtils, ISettingsRepository<Gen
 
             MouseHighlighterSettingsConfig = mouseHighlighterSettingsRepository.SettingsConfig;
             string leftClickColor = MouseHighlighterSettingsConfig.Properties.LeftButtonClickColor.Value;
-            _highlighterLeftButtonClickColor = !string.IsNullOrEmpty(leftClickColor) ? leftClickColor : "#FFFF00";
+            _highlighterLeftButtonClickColor = !string.IsNullOrEmpty(leftClickColor) ? leftClickColor : "#41FFFF00";
 
             string rightClickColor = MouseHighlighterSettingsConfig.Properties.RightButtonClickColor.Value;
-            _highlighterRightButtonClickColor = !string.IsNullOrEmpty(rightClickColor) ? rightClickColor : "#0000FF";
+            _highlighterRightButtonClickColor = !string.IsNullOrEmpty(rightClickColor) ? rightClickColor : "#410000FF";
+
+            string alwaysColor = MouseHighlighterSettingsConfig.Properties.AlwaysColor.Value;
+            _highlighterAlwaysColor = !string.IsNullOrEmpty(rightClickColor) ? rightClickColor : "#00FF0000";
```

Comment:
I've observed "always color" to be the same as one of the other some times. I think this might be the reason.

Reply:
Thank you for finding this bug! I completely overlooked it.
I fixed it.